server/chat-commands/admin.ts

```diff
@@ -1102,3 +1121,28 @@ export const commands: ChatCommands = {
 		`[player] must be a username or number, [pokemon] must be species name or party slot number (not nickname), [move] must be move name.`,
 	],
 };
+
+export const pages: PageTable = {
+	bot(args, user) {
+		const [bot, pageid] = args;
+		const targetBot = Users.get(bot);
```

Comment:
```suggestion
		const [botid, pageid] = args;
		const bot = Users.get(botid);
```

That's not what `target` means...